app/styles/layout.scss

```diff
@@ -80,6 +78,27 @@ md-input-container {
   width: 100%;
 }
 
+.status {
+  align-items: center;
+  border: 2px solid rgba(55, 71, 79, 0.36);
+  box-shadow: 2px 2px 2px #607d8b;
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  margin: 0;
+  width: 100%;
+
+
+  md-icon.md-default-theme {
```

Comment:
this tells me that `md-icon` is an attribute, is that correct? I dont know of any html that looks like that :)